gkShell/config/mariadb/TODO.md

```diff
@@ -0,0 +1,24 @@
+## Registration
+In order to create an user account, we need to update following db fields: 
+
+`gk-users` table : need to allow NULL value
+
+- [ ] Error No 1364: Field 'email' doesn't have a default value
+- [ ] Error No 1364: Field 'ostatni_login' doesn't have a default value
+
+
+## Others
+(need to double check that)
+
+## Geokrety
+
+- Error No 1364: Field 'droga' doesn't have a default value
```

Comment:
Relates to #220 too

Reply:
yep TODO / I have to make this changes/ will do a new PR as 220 is merged

Reply:
I move TODO as new Issue #226